azurerm/resource_arm_automation_job_schedule_test.go

```diff
@@ -0,0 +1,315 @@
+		if !hasResourceGroup {
+			return fmt.Errorf("Bad: no resource group found in state for Automation Job Schedule: '%s'", jobScheduleUUID)
+		}
+
+		resp, err := conn.Get(ctx, resourceGroup, accName, jobScheduleUUID)
+
+		if err != nil {
+			if utils.ResponseWasNotFound(resp.Response) {
+				return nil
+			}
+
+			return err
+		}
+
+		return fmt.Errorf("Automation Job Schedule still exists:\n%#v", resp)
+	}
+
+	return nil
+}
+
+func testCheckAzureRMAutomationJobScheduleExists(resourceName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		conn := testAccProvider.Meta().(*ArmClient).automationJobScheduleClient
+		ctx := testAccProvider.Meta().(*ArmClient).StopContext
+
+		// Ensure we have enough information in state to look up in API
+		rs, ok := s.RootModule().Resources[resourceName]
+		if !ok {
+			return fmt.Errorf("Not found: %s", resourceName)
+		}
+
+		id, err := parseAzureResourceID(rs.Primary.Attributes["id"])
+		if err != nil {
+			return err
+		}
+		jobScheduleID := id.Path["jobSchedules"]
+		jobScheduleUUID := uuid.FromStringOrNil(jobScheduleID)
+		accName := rs.Primary.Attributes["automation_account_name"]
+
+		resourceGroup, hasResourceGroup := rs.Primary.Attributes["resource_group_name"]
+		if !hasResourceGroup {
+			return fmt.Errorf("Bad: no resource group found in state for Automation Job Schedule: '%s'", jobScheduleUUID)
+		}
+
+		resp, err := conn.Get(ctx, resourceGroup, accName, jobScheduleUUID)
+		if err != nil {
+			if utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("Automation Job Schedule '%s' (Account %q / Resource Group %q) does not exist", jobScheduleUUID, accName, resourceGroup)
+			}
+
+			return fmt.Errorf("Bad: Get on automationJobScheduleClient: %+v", err)
+		}
+
+		return nil
+	}
+}
+
+func testAccAzureRMAutomationJobSchedulePrerequisites(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_automation_account" "test" {
+  name                = "acctestAA-%d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+
+  sku {
+    name = "Basic"
+  }
+}
+
+resource "azurerm_automation_runbook" "test" {
+  name                = "Output-HelloWorld"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+
+  account_name = "${azurerm_automation_account.test.name}"
+  log_verbose  = "true"
+  log_progress = "true"
+  description  = "This is a test runbook for terraform acceptance test"
+  runbook_type = "PowerShell"
+
+  publish_content_link {
+    uri = "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/101-automation-runbook-getvms/Runbooks/Get-AzureVMTutorial.ps1"
+  }
+
+  content = <<EOF
+  param(
+    [string]$Output = "World"
```

Comment:
Hi @katbyte, if the test parameters are added, then they probably should be added to the runbook here as well.

Reply:
👍 

fwiw, i'm getting the same weird casing issue, except some that change for you stay the same here 😅